File: 2265-partition-array-according-to-given-pivot/partition-array-according-to-given-pivot.py
class Solution:
    def pivotArray(self, nums: List[int], pivot: int) -> List[int]:
        p=[]
        r=[]
        q=[]
        for i in range(len(nums)):
            if(nums[i]<pivot):
                p.append(nums[i])
            elif(nums[i]>pivot):
                q.append(nums[i])
            else:
                r.append(nums[i])
        return p+r+q  

File: 2265-partition-array-according-to-given-pivot/partition-array-according-to-given-pivot.cpp
class Solution {
public:
    vector<int> pivotArray(vector<int>& nums, int pivot) {
        vector<int>p,q,r;
        for(int i=0;i<nums.size();i++)
        {
            if(nums[i]<pivot) p.push_back(nums[i]);
            else if(nums[i]>pivot) q.push_back(nums[i]);
            else r.push_back(nums[i]);
        }
        copy(r.begin(), r.end(), back_inserter(p));
        copy(q.begin(), q.end(), back_inserter(p));
        nums.clear();
        for(int i=0;i<p.size();i++)
        {
            nums.push_back(p[i]);
        }
        return nums;
    }
};